src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -14,194 +21,107 @@ import (
 	"github.com/twmb/types"
 )
 
-func newListPartitionMovementsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+func newListCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	var (
-		completion       int
-		all              bool
-		human            bool
-		partitions       []string
-		response         []adminapi.ReconfigurationsResponse
-		filteredResponse []adminapi.ReconfigurationsResponse
+		disabledOnly  bool
+		withInternals bool
 	)
 	cmd := &cobra.Command{
-		Use:   "move-status",
-		Short: "Show ongoing partition movements",
-		Long:  helpListMovement,
+		Use:     "list [TOPICS]",
+		Aliases: []string{"ls"},
+		Short:   "List partitions in the cluster",
+		Long: `List partitions in the cluster
+
+This commands lists the cluster-level metadata of all partitions in the cluster, 
+you may filter the topics you want to list by passing them as argument to this
+command.
+
+The REPLICAS column displayed in the output table contains a list of replicas
+assignments in the form of: <Node-ID>:<Core>.
+
+EXAMPLES
+
+List all partitions in the cluster.
+  rpk cluster partitions list
+
+List all partitions in the cluster, filtering for topic foo and bar.
+  rpk cluster partitions list foo bar
+
+List only disabled partitions.
+  rpk cluster partitions list --disabled
+
+List all in json format, but include also internal topics.
+  rpk cluster partition list --format json --with-internals
+`,
 		Run: func(cmd *cobra.Command, topics []string) {
+			f := p.Formatter
+			if h, ok := f.Help([]adminapi.ClusterPartition{}); ok {
+				out.Exit(h)
+			}
 			p, err := p.LoadVirtualProfile(fs)
 			out.MaybeDie(err, "unable to load config: %v", err)
 			out.CheckExitCloudAdmin(p)
 
-			// If partition(s) is specified but no topic(s) is specified, exit.
-			if len(topics) <= 0 && len(partitions) > 0 {
-				out.Die("specify at least one topic when --partition is used, exiting.")
-			}
-
 			cl, err := adminapi.NewClient(fs, p)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
-			response, err = cl.Reconfigurations(cmd.Context())
-			out.MaybeDie(err, "unable to list partition movements: %v\n", err)
-
-			if len(response) == 0 {
-				out.Exit("There are no ongoing partition movements.")
-			}
-
-			for _, t := range topics {
-				nt := strings.Split(t, "/")
-				if len(nt) > 2 {
-					fmt.Printf("invalid format for topic %s, skipping.\n", t)
-					continue
+			var partitions []adminapi.ClusterPartition
+			if len(topics) == 0 {
+				partitions, err = cl.AllClusterPartitions(cmd.Context(), withInternals, disabledOnly)
+				out.MaybeDie(err, "unable to query all partitions in the cluster: %v", err)
+			} else {
+				if withInternals {
+					out.Die("--with-internals not available when listing single topics.\n%v", cmd.UsageString())
```

Comment:
I'm experimenting here with the error message, printing the error + usage after an incorrect use of flags and/or args is a better alternative to just printing the error.

However, the usageString does not contain the long text, so, wdyt of adding something like `out.ExecAndDie`, we print a message, exec, and die. That way we can:
```
  out.ExecAndDie(cmd.Help, "my message %v", args)
```

But not sure...